staging/src/kubevirt.io/api/instancetype/v1beta1/types.go

```diff
@@ -116,6 +116,11 @@ type VirtualMachineInstancetypeSpec struct {
 	//
 	// +optional
 	LaunchSecurity *v1.LaunchSecurity `json:"launchSecurity,omitempty"`
+
+	// Optionally defines the Required Annotations to be used by the instancetype.
```

Comment:
`to be used by the instance type and applied to the VirtualMachine and VirtualMachineInstance`?